specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1438,17 +1438,206 @@
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/NorthSouthHardenings"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{extendedResourceProvider}/{extendedResourceType}/{extendedResourceName}/providers/Microsoft.Security/northSouthHardenings/{northSouthResourceName}/{trafficHardeningsRulesEnforceAction}": {
+            "post": {
+                "x-ms-examples": {
+                    "Enforces the given collections of traffic hardenings rule's on the VM's NSG": {
+                        "$ref": "./examples/NorthSouthHardenings/EnforceTrafficHardeningRules_example.json"
+                    }
+                },
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "description": "Enforces the given collections of traffic hardenings rule's on the VM's NSG",
+                "operationId": "NorthSouthHardenings_Enforce",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "$ref": "#/parameters/ExtendedResourceProvider"
+                    },
+                    {
+                        "$ref": "#/parameters/ExtendedResourceType"
+                    },
+                    {
+                        "$ref": "#/parameters/ExtendedResourceName"
+                    },
+                    {
+                        "$ref": "#/parameters/NorthSouthResourceName"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    },
+                    {
+                        "$ref": "#/parameters/TrafficHardeningsRulesEnforceAction"
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "Accepted"
+                    },
+                    "204": {
+                        "description": "No Content"
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },

```

Comment:
This is an async operation, but I don't see the [x-ms-long-running-operation ](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation) extension